lib/ruby_warning_filter.rb

```diff
@@ -20,33 +20,39 @@
 #
 class RubyWarningFilter < DelegateClass(IO)
   attr_reader :ruby_warnings
+  BACKTRACE = "\tfrom"
+  NEWLINE = "\n"
```

Comment:
You can keep string literals inline with `# frozen_string_literal: true` comment directive.

Reply:
👍

Reply:
Thanks, but I still prefer to keep all conditions in named constants for readability & consistency.